main.js

```diff
@@ -1 +1,18 @@
-console.log('Hello World');
\ No newline at end of file
+const level = require('level');
+const db = connectToDatabase('./studentdb');
+
+function connectToDatabase(db){
+    return level(db, { valueEncoding: 'json' });
+}
+
+var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probationary'];
+
+acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
```

Comment:
in preparation for next task, make acceptStudent function as async function , then wrap this line in a function like
```suggestion
(async function() {
     await acceptStudent('201812485', 'Abdul Moiz Solaiman', 22, 'Marawi City');
}());```

Reply:
noted sir.